### ISSUE TITLE
No incomming call screen on Huawei P40pro

### ISSUE THREAD

**Issue by gpopesc** [nextcloud/talk-android#918]
On my P40pro, when somebody call me, I don't see the call screen to answer the call. Sometimes with screen open, I can see the pop notification and I can press on it and it shows the incoming screen (but this is very rare and random). Also, I don't see any messages notifications.
Moreover, I have a push notification tester installed and when I run it it shows the notification from talk, long time after I hang up the call.

Mentions:
- I have google services installed and push notification are working
- I have Android 10
- I had disabled battery optimization for talk
- I have TURN server
- I have tried the version 9 and 8.0.10, with same result.

I have other 2 devices: Huawei matepad pro (with push notification activated for google) and a P30 pro and everything is working fine: talk popup on locked screen and I can answer without any problems with video or voice..
In 2 PC and  I don't have any problems, everything is working fine.

**Comment by ringogalga** [comment]
same problem with Oneplus 7 and Android 10

**Comment by gpopesc** [comment]
The problem is with push notifications. Somehow, for unknown reason I had received push notifications with delay. I noticed that in other apps using the same service. 

I had to reset to factory my phone and now everything is working fine.

The best way to test your push notification server is with the app below and schedule to receive a push notification after 15 minute or more.
https://play.google.com/store/apps/details?id=com.firstrowria.pushnotificationtester&hl=ro
